src/pages/dev/send-tokens/interchain-tokens.mdx

```diff
@@ -3,23 +3,45 @@ import { Callout } from "/src/components/callout"
 
 Interchain Tokens are created and managed by our new  Interchain Token Service, which allows you to send tokens cross-chain, build your own asset bridges, build asset transfers into your interchain dApp, and take on many other use cases.
 
 Depending on whether or not you have an existing ERC-20 token, there are two main paths available to you.
 
-1. [I want a new Interchain Token](#new-interchain-tokens)
-1. [I want to upgrade existing token(s)](#making-existing-tokens-interchain)
+- [I want to create a new Interchain Token](#create-a-new-interchain-token)
 
-## New Interchain Tokens
+    a. [I want to register and create a new Interchain Token using the Interchain Token Portal](#create-a-new-interchain-token) 
+    
+    b. [I want to create a new custom Interchain Token(s)](#create-a-new-interchain-token)
+    
+- [I want to upgrade an existing token(s)](#making-existing-tokens-interchain)
 
-### Standardized Tokens
-The simplest type of Interchain Token is to create a brand new [Standardized](#standardized) ERC-20 that is available on multiple chains.
+    a. [I want to make a token on one chain into an Interchain Token](#making-existing-tokens-interchain)
+    
+    b. [I want to link deployed tokens on multiple chains into Interchain Tokens](#making-existing-tokens-interchain)
 
-The first step is to visit the [Interchain Portal](https://testnet.interchain.axelar.dev). Connect your wallet, select a source network where you have funds and choose to deploy a new ERC-20 token. Make sure to choose an initial supply, or choose the advanced option to allow later minting. You can then choose any additional chains for your new token. 
 
-Congratulations! You now have an ERC-20 token available on multiple chains. You can interact with your token on any chain identically, as they are all ERC-20 tokens. You can send, transfer, approve, and use all of the methods you are used to. Additionally, each of these tokens is a [Standardized Interchain Token](https://github.com/axelarnetwork/interchain-token-service/blob/main/contracts/interfaces/IStandardizedToken.sol) and has an `interchainTransfer` method allowing you to transfer it between blockchains.
+## Create a new Interchain Token
+
+### Register and create a new Interchain Token(s) using the Interchain Token Portal
+
+The simplest type of Interchain Token is to create a brand new [standardized](../reference/glossary/#standardized) ERC-20 that is available on multiple chains. [Here is a step-by-step tutorial for creating an Interchain Token using the Interchain Token Portal in 4 steps](https://axelar.network/blog/how-to-create-an-interchain-token-with-axelar-in-4-steps).
+
+- Visit the [Interchain Portal](https://testnet.interchain.axelar.dev)
```

Comment:
switch to `*` instead of - for standardization across the docs

Reply:
Nevermind. Prettier always chooses `-` with no option to change, so I guess we're moving to hyphen everywhere.